### ISSUE TITLE
Not work to open hamburger-menu on mobile size.

### ISSUE THREAD

**Issue by StewEucen** [YukiKAJIHARA/xninja-sample-projects#1]
# Overview

* Hamburger-menu on mobile size did not work.

* Is it unimplemented?

# Commit

* https://github.com/YukiKAJIHARA/xninja-sample-projects/tree/xninja-bulma-sample-project(cdn)

**Comment by YukiKAJIHARA** [comment]
A hamburger is too delicious to implement, sorry for that.

**Comment by YukiKAJIHARA** [comment]
This code has to be added.

https://bulma.io/documentation/components/navbar/

```
document.addEventListener('DOMContentLoaded', () => {

  // Get all "navbar-burger" elements
  const $navbarBurgers = Array.prototype.slice.call(document.querySelectorAll('.navbar-burger'), 0);

  // Check if there are any navbar burgers
  if ($navbarBurgers.length > 0) {

    // Add a click event on each of them
    $navbarBurgers.forEach( el => {
      el.addEventListener('click', () => {

        // Get the target from the "data-target" attribute
        const target = el.dataset.target;
        const $target = document.getElementById(target);

        // Toggle the "is-active" class on both the "navbar-burger" and the "navbar-menu"
        el.classList.toggle('is-active');
        $target.classList.toggle('is-active');

      });
    });
  }

});

```

**Comment by YukiKAJIHARA** [comment]
The issue was solved only on "x-ninja-bulma-without-npm" branch.
Does it work correctly?

**Comment by YukiKAJIHARA** [comment]
This issue was fixed in v1.1 on all branches.

**Comment by StewEucen** [comment]
Thank you for my tallest buddy!

This issue has done.
